food_server/app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask
+from flask import request
+import storage
+app = Flask(__name__)
+
+
+@app.get('/food')
```

Comment:
Would be better to do only one route for the same urls.. and devide functionality in this function like:
```python
@app.route('/food')
def food_view():
    if request.method == 'post' than ...
    elif request.method == 'get' than ...
```